e2e-tests/development-runtime/package.json

```diff
@@ -32,7 +32,7 @@
   "license": "MIT",
   "scripts": {
     "build": "gatsby build",
-    "develop": "cross-env CYPRESS_SUPPORT=y ENABLE_GATSBY_REFRESH_ENDPOINT=true GATSBY_EXPERIMENTAL_QUERY_ON_DEMAND=y gatsby develop",
+    "develop": "cross-env CYPRESS_SUPPORT=y ENABLE_GATSBY_REFRESH_ENDPOINT=y GATSBY_ENABLE_QOD_IN_CI=y gatsby develop",
```

Comment:
I was confused what `GATSBY_ENABLE_QOD_IN_CI` did for some time while reviewing this.

We should probably avoid adding new acronyms to the vernacular where possible (more searchable, accessible to newcomers, etc.), even if the alternatives are more verbose.

Maybe something like `GATSBY_ENABLE_QUERY_ON_DEMAND_IN_CI` or `GATSBY_QUERY_ON_DEMAND_CI`?